### PR TITLE
SSCS-4106: Evidence received Appellant notification (paper case)

### PR DESCRIPTION
**Pull request by ryandac**
Evidence received (not attending a hearing) | Appellant notification

### CODE REVIEW

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            throws Exception {
+        //Given
+        final String appellantEmailId = getFieldValue(appealHearingType, "AppellantEmailId");
+        final String appellantSmsId = getFieldValue(appealHearingType, "AppellantSmsId");
+        final String repsEmailId = getFieldValue(appealHearingType, "RepsEmailId");
+        final String repsSmsId = getFieldValue(appealHearingType, "RepsSmsId");
+
+        simulateCcdCallback(EVIDENCE_RECEIVED_NOTIFICATION,
+                "representative/"
+                        + (appealHearingType != null ? (appealHearingType.name().toLowerCase() + "-") : "")
```
Comment (oleg-hmcts) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java]: Check for null could be removed.

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+        simulateCcdCallback(EVIDENCE_RECEIVED_NOTIFICATION,
+                "representative/" + "no-reps-subscribed-"
+                        + appealHearingType.name().toLowerCase() + "-"
+                        + EVIDENCE_RECEIVED_NOTIFICATION.getId() + "Callback.json");
+
+        List<Notification> notifications = tryFetchNotificationsForTestCase(appellantEmailId,
+                    appellantSmsId);
+        assertNotificationBodyContains(notifications, appellantEmailId);
+        assertNotificationBodyContains(notifications, appellantSmsId);
```
Comment (oleg-hmcts) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java]: The above 4 lines duplicate the appellant test above.

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        List<Notification> notificationsNotFound = tryFetchNotificationsForTestCaseWithFlag(true,
```
Comment (oleg-hmcts) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java]: This variable name is misleading.

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WithRepresentativePersonalisationTest.java [hunk 4]
```diff
@@ -41,19 +37,6 @@
...
-    @Value("${notification.evidenceReceived.appellant.smsId}")
-    private String evidenceReceivedAppellantSmsId;
-    @Value("${notification.paper.evidenceReceived.representative.emailId}")
-    private String paperEvidenceReceivedRepsEmailId;
-    @Value("${notification.evidenceReceived.representative.smsId}")
-    private String paperEvidenceReceivedRepsSmsId;
-    @Value("${notification.oral.evidenceReceived.representative.emailId}")
-    private String oralEvidenceReceivedRepsEmailId;
-    @Value("${notification.evidenceReceived.representative.smsId}")
-    private String oralEvidenceReceivedRepsSmsId;
```
Comment (oleg-hmcts) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WithRepresentativePersonalisationTest.java]: There are also duplicate lines for evidenceReceived in NotificationsFunctionalTest

**Comment by oleg-hmcts** [comment]
LGTM, there are some small readability comments.

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java (hunk 3 above)
Comment (ryandac) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java]: it makes sense to me. how is it misleading?

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WithRepresentativePersonalisationTest.java (hunk 4 above)
Comment (ryandac) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WithRepresentativePersonalisationTest.java]: The lines are removed.

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java (hunk 3 above)
Comment (oleg-hmcts) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java]: Is this not a list of found notifications?

Comment (ryandac) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/EvidenceReceivedFunctionalTest.java]: No. They shouldn't be found. (btw: I didn't write this).

**Comment by mallikarjunputtana** [comment]
Passed testing in Local, Jenkins build is as well green

**Pull request merged**